docs/source/getting_started/index.ipynb

```diff
@@ -1204,7 +1200,7 @@
    ],
    "source": [
     "multiyear_tbl = src.load(\"FIELD CONTACTS\", year = [2021, 2024])\n",
-    "multiyear_tbl"
+    "multiyear_tbl.table.groupby(\"Year\").size()"
```

Comment:
To provide more clarity on the output, I updated this line to show the summarized content of the table using size() to help users understand the size of their data by each year, and also to validate that the year is present on the data

Reply:
This is a good addition!